docker-compose.yml

```diff
@@ -3,6 +3,9 @@ version: "3"
 services:
   autorest-gen: 
     image: reg-hk.agodadev.io/library/autorest-gen:${IMAGE_VERSION}
+    # build: .
+    # volumes:
+    #   - /Users/nkar/workspaces/temp/autorest/output:/app/output
```

Comment:
Can you remove these?

Reply:
It is helpful for local testing.

Reply:
If you want I can put some more comments to make it more readable.

Reply:
done. Added more comment as suggested.

Reply:
Yes please add some comments in there